md_api/src/main/java/com/wipro/ats/bdre/md/api/ArrangePositions.java

```diff
@@ -132,7 +125,8 @@ public ArrangePositions() {
                 String[] nextProcesses = p.getNextProcessIds().split(",");
                 List<PositionsInfo> children = new ArrayList<PositionsInfo>();
                 for (String nextProcess : nextProcesses) {
-                    if (nextProcess.equals("0") || nextProcess.equals(processId.toString())) continue;
+                    if ("0".equals(nextProcess) || nextProcess.equals(processId.toString()))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)